cogs/gaminator.py

```diff
@@ -122,7 +122,7 @@ class Gaminator(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
-    @commands.command(name='other-games')
+    @commands.command(name='other-games', aliases=['og'])
```

Comment:
Ah yep, to be honest 'other-games' was a placeholder but if you're happy with the name, I'm happy.